OneSignalSDK/onesignal/core/src/main/java/com/onesignal/common/services/ServiceProvider.kt

```diff
@@ -80,6 +85,26 @@ class ServiceProvider(
         }
     }
 
+    fun scheduleStartServices() {
+        synchronized(serviceMap) {
+            for (serviceReg in serviceMap!![IStartableService::class.java]!!) {
+                // we can directly start services that were previously resolved
+                if (serviceReg.isResolved()) {
```

Comment:
Why not just use getService() to simply this?

Reply:
Yes the newest commit simplifies the code and now directly calls getService.